karma.conf.js

```diff
@@ -41,14 +41,15 @@ module.exports = (config) => {
     files: [
       'test/index.js',
       { pattern: 'public/fonts/icomoon.ttf', served: true, included: false },
-      { pattern: 'public/images/*', served: true, included: false },
```

Comment:
I realized this folder wasn't being used anymore (since the entities-intro.svg) got removed and there was probably a test warning about this before.